### PR TITLE
Rename vso to codespaces

### PR DESCRIPTION
**Pull request by taiyosogawa**
I ran install, audit, and test, and everything looked fine. As soon as I merge this PR, I'll update the repo to microsoft/homebrew-codespaces

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 **Homebrew tap for Visual Studio Online.**
```
Comment (derekbekoe) [inline README.md:1]: Codespaces

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-You can now run the `vso start` to get started.
+You can now run the `codespaces start` to get started.
 
 To upgrade to the latest version, run:
 
 ```
-brew upgrade vso
+brew upgrade codespaces
 ```
```
Comment (derekbekoe) [inline README.md:14]: On line 17, the "Issues can be reported" url should be https://github.com/MicrosoftDocs/vscodespaces/issues and link text should be MicrosoftDocs/vscodespaces.

File: README.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
 **Homebrew tap for Visual Studio Online.**
 
 ```
-brew install microsoft/vsonline/vso
+brew install microsoft/vsonline/codespaces
```
Comment (derekbekoe) [inline README.md:4]: microsoft/vsonline/codespaces -> microsoft/vscodespaces/codespaces

Comment (taiyosogawa) [reply README.md]: thanks! done

File: README.md (hunk 2 above)
Comment (taiyosogawa) [reply README.md:14]: done!

File: README.md (hunk 1 above)
Comment (taiyosogawa) [reply README.md]: done!

**Pull request merged**